### PR TITLE
 Separate in-memory cache from storage drivers.

### PR DESCRIPTION
**Pull request by rjnagal**

### CODE REVIEW

File: cache/cache.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import info "github.com/google/cadvisor/info/v1"
+
+type StorageDriver interface {
```
Comment (vmarmol) [inline cache/cache.go]: WDYT of renaming this to Cache? It may be confusing to have two storage drivers.

**Comment by vmarmol** [comment]
LGTM

File: cache/cache.go (hunk 1 above)
Comment (rjnagal) [inline cache/cache.go]: fixed.

**Comment by vmarmol** [comment]
LGTM, thanks @rjnagal!

**Pull request merged**